components-core/src/main/java/com/adyen/checkout/components/core/PaymentMethodTypes.kt

```diff
@@ -73,6 +73,7 @@ object PaymentMethodTypes {
     const val BOLETOBANCARIO_HSBC = "boletobancario_hsbc"
     const val BOLETOBANCARIO_ITAU = "boletobancario_itau"
     const val BOLETOBANCARIO_SANTANDER = "boletobancario_santander"
+    const val BOLETO_PRIMEIRO_PAY = "primeiropay_boleto"
```

Comment:
These should not be here in this PR

Reply:
this is here as I need to add it in `VoucherPaymentMethodConfig`

Reply:
I removed support for Boleto in the previous PR and added everything here.